go.mod

```diff
@@ -1,28 +1,25 @@
 module github.com/go-surreal/sdbc
 
-go 1.22.7
+go 1.24
```

Comment:
The project aims to support the latest two "major" (in go that is basically minor) versions of go, so as of now we should require go 1.23(.7) at most.